package.json

```diff
@@ -40,7 +40,7 @@
     },
     "overrides": [
       {
-        "files": "**/__tests__/**/*.spec.ts",
+        "files": ["**/__tests__/**/*.spec.ts", "**/__tests__/**/*.ava.ts"],
```

Comment:
Is the idea here that we do the following?

1. Publish near-runner-ava
2. Update & publish near-runner-jest with a deprecation message at the top of the README
3. Remove near-runner-jest and associated tests from this repository

Reply:
Yep.  Sounds about right.